src/Medusa.ts

```diff
@@ -45,11 +46,30 @@ export class Medusa {
 	 * @param modules The modules to load into medusa
 	 */
 	public async load(modules: Type[]): Promise<MedusaContainer> {
-		const { configModule } = getConfigFile(this.#rootDir, 'medusa-config') as {
+		const configuration = getConfigFile(process.cwd(), `medusa-config`) as {
+			configModule: ConfigModule;
+			configFilePath: string;
+		};
+		const resolveConfigProperties = async (obj): Promise<ConfigModule> => {
```

Comment:
todo: should be moved to utilities and if you can add a new unit test for it please

Reply:
moved to utilities, need to add a unit test

Reply:
added unit test